### PR TITLE
Added connectWithConfig.

### PR DESCRIPTION
**Pull request by guptaamol**
Added connectWithConfig to Specify 'groupOwnerIntent' to set Group Owner While Connecting.

### CODE REVIEW

**Comment by kirillzyusko** [comment]
Hi @guptaamol
Could you please add some info about this method in README.MD?

**Comment by guptaamol** [comment]
@kirillzyusko  I Have updated README.md please do take a look. Also there are some issues with respect to permissions for Android 10, one needs `ACCESS_FINE_LOCATION` instead of `ACCESS_COARSE_LOCATION` to make things work, i guess that needs to be a separate issue.

File: README.MD [hunk 1]
```diff
@@ -223,6 +223,19 @@ connect('22:47:da:9d:58:83')
...
 
+
+### connectWithConfig(connectionArgs)
+
+Connect to network. The method takes two arguments `deviceAddress` and an optional 
+`groupOwnerIntent`. `groupOwnerIntent` specifies to the OS the intentsity with which the device initiaiting the connection wants to be the group Owner.(i.e the server). It takes a value between `0` and `15`, 0 and 15 being the least and most likely group owners respectively. i.e device specifying `groupOwnerIntent 15` is more likely to be a group owner.
+Link to andrdoid documentation [https://developer.android.com/reference/android/net/wifi/p2p/WifiP2pConfig]
+
+```javascript
+connect({ deviceAddress: '22:47:da:9d:58:83', groupOwnerIntent: 15 })
```
Comment (kirillzyusko) [inline README.MD:234]: Shouldn't it be `connectWithConfig ` ? :)

**Review by kirillzyusko — CHANGES_REQUESTED** [review]

**Comment by kirillzyusko** [comment]
@guptaamol Thank you!

> one needs ACCESS_FINE_LOCATION instead of ACCESS_COARSE_LOCATION to make things work

I think it's only Android 10 specific, because on Android 8 & 9 everything worked fine. In any case, thank you for pointing it out. As soon as you PR will be merged I'll change the info about permissions and will publish a new version.

**Comment by guptaamol** [comment]
Your right my bad 😅

File: README.MD (hunk 1 above)
Comment (guptaamol) [reply README.MD]: Fixed It.

**Comment by kirillzyusko** [comment]
Published as 3.2.0

**Pull request merged**